rfc/5/versions/1/index.md

```diff
@@ -0,0 +1,977 @@
+
+Conforming readers:
+- MUST parse `identity`, `scale`, `translation` transformations;
+- SHOULD parse `mapAxis`, `affine`, `rotation` transformations;
+- SHOULD display an informative warning if encountering transformations that cannot be parsed;
+- SHOULD be able to apply transformations to points;
+- SHOULD be able to apply transformations to images;
+
+Coordinate transformations can be stored in multiple places to reflect different usecases.
+     
+- Multiscale transformations represent a special case of transformations
+  and are explained [below](#multiscales-metadata).
+- Additional transformations for single images MUST be stored under a field `coordinateTransformations`
+  in the multiscales dictionaries.
+  This `coordinateTransformations` field MUST contain a list of valid [transformations](#transformation-types).
+- Transformations between two or more images MUST be stored in the attributes of a parent zarr group.
+  For transformations that store data or parameters in a zarr array,
+  those zarr arrays SHOULD be stored in a zarr group `coordinateTransformations`.
+
+Implementations SHOULD prefer to store transformations as a sequence of less expressive transformations
+(i.e., sequence[translation, rotation] instead of affine transformation with translation/rotation) component. 
+
+<pre>
+store.zarr                      # Root folder of the zarr store
+│
+├── zarr.json                   # coordinate transformations describing the relationship between two image coordinate systems
+│                               # are stored in the attributes of their parent group.
+│                               # transformations between 'volume' and 'crop' coordinate systems are stored here.
+│
+├── coordinateTransformations   # transformations that use array storage go in a "coordinateTransformations" zarr group.
+│   └── displacements           # for example, a zarr array containing a displacement field
+│       └── zarr.json
+│
+├── volume
+│   ├── zarr.json              # group level attributes (multiscales)
+│   └── 0                       # a group containing the 0th scale
+│       └── image               # a zarr array
+│           └── zarr.json      # physical coordinate system and transformations here
+└── crop
+    ├── zarr.json              # group level attributes (multiscales)
+    └── 0                       # a group containing the 0th scale
+        └── image               # a zarr array
+            └── zarr.json      # physical coordinate system and transformations here
+</pre>
+
+````{admonition} Example
+Two instruments simultaneously image the same sample from two different angles,
+and the 3D data from both instruments are calibrated to "micrometer" units.
+Two samples are collected ("sampleA" and "sampleB").
```

Comment:
```suggestion
```
sampleB is never again mentioned after here, so I think fine to delete this sentence?